tests/experimental/test_experimental_gaussian_process_exact.py

```diff
@@ -0,0 +1,75 @@
+import gpytorch
+import pytest
+import torch
+from autoemulate.experimental.emulators.gaussian_process.exact import (
+    GaussianProcessExact,
+)
+from autoemulate.emulators.gaussian_process import constant_mean, rbf, rbf_times_linear
+from autoemulate.experimental.tuner import Tuner
+from autoemulate.experimental.types import DistributionLike
+from sklearn.datasets import make_regression
+
+
+@pytest.fixture
+def sample_data_y1d():
+    x, y = make_regression(n_samples=20, n_features=5, n_targets=1, random_state=0)  # type: ignore
+    return torch.Tensor(x), torch.Tensor(y)
+
+
+@pytest.fixture
+def new_data_y1d():
+    x, y = make_regression(n_samples=20, n_features=5, n_targets=1, random_state=1)  # type: ignore
+    return torch.Tensor(x), torch.Tensor(y)
+
+
+@pytest.fixture
+def sample_data_y2d():
+    x, y = make_regression(n_samples=20, n_features=5, n_targets=2, random_state=0)  # type: ignore
+    return torch.Tensor(x), torch.Tensor(y)
+
+
+@pytest.fixture
```

Comment:
Perhaps move these data functions to a separate place where they can be easily imported by other test files for other models - just googled and looks like for pytest you should create a `conftest.py`: https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session

Reply:
Thanks @edwardchalstrey1, this sounds great.

Reply:
Added in [5aeee8d](https://github.com/alan-turing-institute/autoemulate/pull/384/commits/5aeee8d7e23457a7b61a4dd0a678dc527808067d)